src/foam/nanos/pii/CapabilityRefine.js

```diff
@@ -0,0 +1,31 @@
+/**
+ * @license
+ * Copyright 2025 The FOAM Authors. All Rights Reserved.
+ * http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+foam.CLASS({
+  package: 'foam.nanos.pii',
+  name: 'CapabilityRefine',
+  refines: 'foam.nanos.crunch.Capability',
+
+  properties: [
+    {
+      /**
+        Identifies properties that contain Personally identifiable information,
```

Comment:
IS this comment corrent?  We're refining onto capability but it talks about properties that contain PII.

I would assume you mean the capability contains PII?

Reply:
This file should have been deleted - and now has been.